automated install/basic-install.sh

```diff
@@ -2304,6 +2304,25 @@ main() {
     fi
   fi
 
+  # resolved and dnsmasq can't share port 53. 
+  # resolved needs to remain in place for installer to download needed files
+  # so this change needs to be made after installation is complete, but before resarting dnsmasq/ftl
+
+  # Check if running ubuntu 18.04 bionic beaver, which ships with resolved active on port 53
+  # (This check may need to be broadened for other systems running resolved?)
+  if ( lsb_release -a | grep 'Ubuntu 18.04' &> /dev/null ); then
```

Comment:
Why is this check needed at all (it feels very hacky)? Couldn't we just check if `systemd` is used as init and then if `systemd-resolved` is enabled?

Reply:
That's intended as a stopgap "please don't break any other distro" measure. 

As systemd continues it's march onwards, this will certainly be an issue in other distros. If they keep /etc/systemd/resolved.conf where it belongs then there shouldn't be a problem with skipping the ubuntu check, but I'd err on the cautious side for the moment.